Microsoft.VisualStudio.MiniEditor/Microsoft.VisualStudio.MiniEditor.csproj

```diff
@@ -220,48 +232,48 @@
     <EmbeddedResource Include="$(BaseEditorPath)Core\Impl\ContentType\Strings.resx">
       <Generator>ResXFileCodeGenerator</Generator>
       <ManifestResourceName>Microsoft.VisualStudio.CoreUtilityImplementation.ContentType.Strings</ManifestResourceName>
-      <LastGenOutput>$(BaseEditorPath)Core\Impl\ContentType\Strings.Designer.cs</LastGenOutput>
+      <LastGenOutput>vs-editor-api\src\Editor\Core\Impl\ContentType\Strings.Designer.cs</LastGenOutput>

```

Comment:
Same for those, is it to make VSMac happy?

Reply:
I didn't make them, it did 😞